bip-PC.md

```diff
@@ -0,0 +1,252 @@
+<pre>
+  BIP: ?
+  Layer: Consensus (soft fork)
+  Title: OP_PAIRCOMMIT
+  Author: moonsettler <moonsettler@protonmail.com>
+  Comments-Summary: No comments yet.
```

Comment:
The Comments-Summary header is optional, and the comment system has fallen out of use. (And will hopefully be sunset in the near future.)

```suggestion
```